### PR TITLE
add gpl2 license

### PR DESCRIPTION
**Pull request by robbyt**
This PR also removes the the footer after the GPL3 license ends.

### CODE REVIEW

File: cobra/cmd/licenses.go [hunk 1]
```diff
@@ -425,6 +425,305 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 `,
 	}
 
+	Licenses["gpl2"] = License{
+		Name:            "GNU General Public License 3.0",
```
Comment (eparis) [inline cobra/cmd/licenses.go]: 3.0 ?

Comment (robbyt) [inline cobra/cmd/licenses.go]: Oops I missed this in the bit that I pasted. I will fix that in a moment.

**Comment by robbyt** [comment]
@eparis sorry I missed the title in the original commit. Does this look ok now?

**Comment by spf13** [comment]
Merged as 3dc021e

**Pull request closed without merging**